dcos_test_utils/dcos_api.py

```diff
@@ -358,7 +358,8 @@ def _wait_for_metronome(self):
             log.info('Continuing to wait for Metronome')
             return False
 
-        assert r.status_code == 200
+        assert r.status_code == 200, "Expecting status code 200 for Metronome but got {} with body {}"\
```

Comment:
Instead, I would prefer that this change is removed, and the log above be optionally changed to use `.content` rather than `.text`. However, that is a personal preference and not a requirement.

Reply:
In particular, with this change, the same response code and body will be logged twice, which I think could be confusing.

Reply:
I strongly believe that assert should provide some meaningful context by itself, ideally every assert in the code should look like that.

When working on Marathon SI tests, I think this practice really helps to make SI tests more easily to debug.

Reply:
Although this is not my preference, you will likely have to debug Marathon problems more than me and so I am happy to defer to your preference 😄

Reply:
If we are not paying per byte printed into the test log I would say it won't be that confusing... :-)